src/util/headers.ts

```diff
@@ -83,14 +84,16 @@ export enum Header {
   XAxiosCacheEtag = 'x-axios-cache-etag',
 
   /**
-   * Used internally to mark the cache item as being revalidatable and enabling stale cache state
-   * may contain 'use-cache-timestamp' if `cache.modifiedSince` is `true`, otherwise will contain a date
-   * from `cache.modifiedSince`.
-   * If a date is provided, it can be used for `If-Modified-Since` header, otherwise the cache timestamp
-   * can be used for `If-Modified-Since` header.
+   * Used internally to mark the cache item as being revalidatable and
+   * enabling stale cache state may contain `'use-cache-timestamp'` if
+   * `cache.modifiedSince` is `true`, otherwise will contain a date
+   * from `cache.modifiedSince`. If a date is provided, it can be used
+   * for `If-Modified-Since` header, otherwise the cache timestamp can
+   * be used for `If-Modified-Since` header.
    *
    * ```txt
    * X-Axios-Cache-Last-Modified: <day-name>, <day> <month> <year> <hour>:<minute>:<second> GMT
+   * X-Axios-Cache-Last-Modified: "use-cache-timestamp"
```

Comment:
actually, quotes won't be here :) (all headers are always string, and for etag, quote must be added)